pkgs/os-specific/linux/firmware/intel2200BGFirmware/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl }:
+
+let version = "3.1"; in
+
+stdenv.mkDerivation {
+  name = "intel2200BGFirmware-${version}";
+
+  src = fetchurl {
+    url = "http://pkgs.fedoraproject.org/repo/pkgs/ipw2200-firmware/ipw2200-fw-3.1.tgz/eaba788643c7cc7483dd67ace70f6e99/ipw2200-fw-${version}.tgz";
```

Comment:
You mean the homepage? <http://ipw2200.sourceforge.net/firmware.php> is alright.